### PR TITLE
Caracal host images

### PR DESCRIPTION
**Pull request by Alex-Welsh**

### CODE REVIEW

**Review by seunghun1ee — DISMISSED** [review]

**Review by markgoddard — DISMISSED** [review]
LGTM if it passes.

**Comment by Alex-Welsh** [comment]
I'm reasonably sure that the RL9 upgrade job will fail due to the selinux change, I'll convert to draft for now

**Review by seunghun1ee — APPROVED** [review]

File: .github/workflows/stackhpc-all-in-one.yml [hunk 1]
```diff
@@ -396,13 +396,15 @@ jobs:
           KAYOBE_AUTOMATION_SSH_PRIVATE_KEY: ${{ steps.ssh_key.outputs.ssh_key }}
         if: inputs.upgrade
 
+      # NOTE(Alex-Welsh): configure host with selinux_state=disabled, because
+      # Antelope images are not built as permissive
```
Comment (MoteHue) [inline .github/workflows/stackhpc-all-in-one.yml:400]: Seems fine for now, can always remove this once we fix the AIOs failing to reboot.

**Review by MoteHue — APPROVED** [review]

**Review by markgoddard — COMMENTED** [review]
woop, selinux

**Pull request merged**